zapisy/apps/offer/proposal/templates/proposal/proposal_info.html

```diff
@@ -1,8 +1,8 @@
 {% load filters %}
 
-<table class="table table-bordered table-md-responsive d-print-none" id="table-info">
+<table class="table table-bordered table-md-responsive d-print-none table-offer" id="table-info">
     <colgroup>
-        <col class="table-info-type"></col>
+        <col class="table-info-type table-what table-info-offer"></col>
```

Comment:
```suggestion
        <col class="table-info-type table-info-offer"></col>
```
Trochę mi to wygląda na pozostałości po edycjach...

Reply:
Przepraszam za niedopatrzenie :) W nowym commicie usunięte